runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -1387,6 +1388,27 @@ TR_J9SharedCache::storeSharedData(J9VMThread *vmThread, const char *key, const J
 #endif
    }
 
+void
+TR_J9SharedCache::getWellKnownClassesSCCKey(char *buffer, size_t size, unsigned int includedClasses)
```

Comment:
I would like a comment describing what this function does. In particular I don't know the meaning of the `includedClasses` parameter.
Not related to this change: I find the name `snprintfNoTrunc` a bit strange given that the buffer is limited in size and the function will truncate to fit the size of the buffer.

Reply:
Also, since this function builds the key rather than retrieving it, maybe it should be called `buildWellKnownClassesSCCKey` or `createWellKnownClassesSCCKey`